[2024-11-11 06:24:06,750: INFO/MainProcess] Task resultsUI.task.process_batch[6eb48b04-7adf-460c-a005-37cd1c597023] received
[2024-11-11 06:24:06,751: WARNING/MainProcess] The current batch id in process is givenas 
[2024-11-11 06:24:06,751: WARNING/MainProcess]  
[2024-11-11 06:24:06,751: WARNING/MainProcess] 4df48c55-88d7-44bd-a5a7-9b2f6882f65f
[2024-11-11 06:24:06,751: WARNING/MainProcess] The code is here in process_batch
[2024-11-11 06:24:06,755: WARNING/MainProcess] Models already downloaded
[2024-11-11 06:24:06,755: WARNING/MainProcess] Already downloaded the checkpoint
[2024-11-11 06:24:06,755: WARNING/MainProcess] link entries are -----------------------------------------------------------------
[2024-11-11 06:24:06,755: WARNING/MainProcess]  
[2024-11-11 06:24:06,757: WARNING/MainProcess] <QuerySet [<LinkEntry: 47fa15a6-4826-46d9-82c7-1e3edf4cb012: https://apivideodata.s3.ap-south-1.amazonaws.com/t.mp4?response-content-disposition=inline&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEBYaCmFwLXNvdXRoLTEiRzBFAiEAk1jlD2a%2B1eSOSt92fvABbaFa6ZMWdBvbs92Jjl9kMFwCIGcbPfBAHKeuZcX18CRSIrjVMTPmws%2FPmr%2B9rG1%2BHUdVKtADCJ%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEQABoMNzMwMzM1MzIwMzAyIgyUfSEy14RzdgxMWgIqpAPxRCrvHDJjdQHx6loBk3BnYVygnxOYTGpfks9h7GhOjeDx3xEq1QSNEA8qK2VmhsO08jUNbOFimVIQRSear6xkFEz8Hyf5HukhTB0Ld7mImSdtG5ra36NWU5MbHIaTOwM48bHYsUWF%2BPo7oimXLz%2B8n5Tkx0B5kiwLFfek6d%2BdzwddF2elc2SID6flYmqnM4FZG6Sqm6dY7udRImpRhWyANkca1nRo2fHvMmlwAi%2BXlguoqVjdM1FJ1E9M3fZodwSR7WlXQwn%2FzOechU6ZbicEvJIx4LylE1a5KmwZrd3mjSmNrn3QvUkYzuq6IzlcQBm%2F3guU1PPGl45yoiFsifJCR3GVQ%2FlyRm03cK7U0a%2Ftj51QOpeSi21QeC%2FINrGEZe%2F4Ph0GHvkMsROE4QSgVyQwSaulXVQ9Qs7lDFwvBWa8UcF7dxkfD6X4lEpyfHhbX8LYyQr77VRRp07pjmpMvaLmn0YVF7N%2BexP59UiMQeEWhPvttWCEHvREDWviIWzxjatYSIfKGZ%2BTIG0iPb6EPGhPUVcn0sVYRzKHpE2%2FgLRhastGEzww1JHGuQY65AInfX8ygsbb8RTp0TMdtf18oYvHxUhvMj%2BS6y3k9kkrnCyjvTZt%2B3u1JyWcYLBMg1GbcYA7duR%2F4eOi%2BieDQ5B%2FBQ4JV2OoMZqXgMJV9K6oCM31eJDOp3EMx%2Ba18jOkCSJXs8fZwK0jfXjoFVhcTsJTNEViRZrWq78%2BCCfChrcWQdoY17mRJVw7IBWMXMaUjaned%2BaM5ZTvxuIdvn2C467QNEZIgASairm93XpZ07vN6AKHqqZnsno0gwnGSeCfXMyei%2BF1EMjt6tYgh%2Bte84E6PYV1MMzxiKHrYechXnwfZW423g54cfzpiNu%2BN1aaQA8qPOqB6rAie3eCjreDUEcFQOH3QJR7RyL0jb0TXe%2FO%2FAZobJSwGcGy%2FC503fOEUVbTiuXBSj21OguBEBsBZ5vHpr4nLv8LLci%2Btloc%2Fn82qhKTm7ESaOLUaBHTt2aVmVpxrRKzL0l1zmAlPNCAZOLINHXRHA%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIA2UC3ATDXJUHLL23Q%2F20241111%2Fap-south-1%2Fs3%2Faws4_request&X-Amz-Date=20241111T062132Z&X-Amz-Expires=43200&X-Amz-SignedHeaders=host&X-Amz-Signature=df0fc96ca7c4413c8b45741ea7401faa3559995e78e65293355ad29d6f5e397f [Pending]>]>
[2024-11-11 06:24:06,766: WARNING/MainProcess] The code is here in command 
[2024-11-11 06:24:08,924: WARNING/MainProcess] Video downloaded and saved to /home/ubuntu/AI_G/videos/4df48c55-88d7-44bd-a5a7-9b2f6882f65f/47fa15a6-4826-46d9-82c7-1e3edf4cb012.mp4
[2024-11-11 06:24:08,925: WARNING/MainProcess] The cnt is **********************************************
[2024-11-11 06:24:08,925: WARNING/MainProcess]  
[2024-11-11 06:24:08,925: WARNING/MainProcess] 1
[2024-11-11 06:24:08,925: WARNING/MainProcess] Current Working Directory is
[2024-11-11 06:24:08,925: WARNING/MainProcess]  
[2024-11-11 06:24:08,925: WARNING/MainProcess] /home/ubuntu/AI_G/resultsUI
[2024-11-11 06:24:08,925: WARNING/MainProcess] The root directory is given as *****************************************************************************************
[2024-11-11 06:24:08,925: WARNING/MainProcess]  
[2024-11-11 06:24:08,925: WARNING/MainProcess] /home/ubuntu/AI_G/resultsUI/
[2024-11-11 06:24:08,929: WARNING/MainProcess] All files deleted successfully from folder: /home/ubuntu/AI_G/resultsUI/datasets/ChaLearn/voice_data/voice_librosa/test_data
[2024-11-11 06:24:08,929: WARNING/MainProcess] All files deleted successfully from folder: /home/ubuntu/AI_G/resultsUI/datasets/ChaLearn/voice_data/voice_raw/test_data
[2024-11-11 06:24:09,088: WARNING/MainProcess] Making frames 
[2024-11-11 06:24:09,090: WARNING/MainProcess]   0%|          | 0/1 [00:00<?, ?it/s]
[2024-11-11 06:24:09,091: WARNING/MainProcess] 100%|##########| 1/1 [00:00<00:00, 6132.02it/s]
[2024-11-11 06:24:09,091: WARNING/MainProcess] Waiting for all subprocesses done...
[2024-11-11 06:24:09,093: WARNING/ForkPoolWorker-9] execute /home/ubuntu/AI_G/resultsUI/datasets/ChaLearn/test/47fa15a6-4826-46d9-82c7-1e3edf4cb012.mp4 ...
[2024-11-11 06:24:31,983: WARNING/ForkPoolWorker-9] Extraction completed, 1796 frames saved.
[2024-11-11 06:24:32,035: WARNING/MainProcess] All subprocesses done.
[2024-11-11 06:24:32,035: WARNING/MainProcess] processed 1 videos
[2024-11-11 06:24:32,038: WARNING/MainProcess] 0it [00:00, ?it/s]
[2024-11-11 06:24:32,038: WARNING/MainProcess] 0it [00:00, ?it/s]
[2024-11-11 06:24:32,039: WARNING/MainProcess] The configuration used is:
[2024-11-11 06:24:32,039: WARNING/MainProcess]  
[2024-11-11 06:24:32,039: WARNING/MainProcess] {'DATA': {'ROOT': 'datasets', 'TYPE': 'frame', 'SESSION': 'talk', 'TRAIN_IMG_DATA': 'ChaLearn/train_data', 'TRAIN_IMG_FACE_DATA': 'image_data/train_data_face', 'TRAIN_AUD_DATA': 'ChaLearn/voice_data/voice_librosa/train_data', 'TRAIN_LABEL_DATA': 'ChaLearn/annotation/annotation_training.pkl', 'VALID_IMG_DATA': 'ChaLearn/valid_data', 'VALID_IMG_FACE_DATA': 'image_data/valid_data_face', 'VALID_AUD_DATA': 'ChaLearn/voice_data/voice_librosa/valid_data', 'VALID_LABEL_DATA': 'ChaLearn/annotation/annotation_validation.pkl', 'TEST_IMG_DATA': 'ChaLearn/test_data', 'TEST_IMG_FACE_DATA': 'image_data/test_data_face', 'TEST_AUD_DATA': 'ChaLearn/voice_data/voice_librosa/test_data', 'TEST_LABEL_DATA': 'ChaLearn/annotation/annotation_test.pkl', 'VA_ROOT': 'datasets', 'VA_DATA': 'va_data/cropped_aligned', 'VA_TRAIN_LABEL': 'va_data/va_label/VA_Set/Train_Set', 'VA_VALID_LABEL': 'va_data/va_label/VA_Set/Validation_Set'}, 'DATA_LOADER': {'NAME': 'bimodal_resnet_data_loader', 'DATASET': '', 'TRANSFORM': 'standard_frame_transform', 'TRAIN_BATCH_SIZE': 8, 'VALID_BATCH_SIZE': 4, 'NUM_WORKERS': 0, 'SHUFFLE': True, 'DROP_LAST': True, 'SECOND_STAGE': {'METHOD': '', 'TYPE': ''}}, 'MODEL': {'NAME': 'audiovisual_resnet', 'PRETRAIN': False, 'NUM_CLASS': 5, 'SPECTRUM_CHANNEL': 50, 'RETURN_FEATURE': False}, 'LOSS': {'NAME': 'mean_square_error'}, 'SOLVER': {'NAME': 'sgd', 'RESET_LR': False, 'LR_INIT': 0.001, 'WEIGHT_DECAY': 0.0005, 'MOMENTUM': 0.9, 'BETA_1': 0.5, 'BETA_2': 0.999, 'SCHEDULER': 'multi_step_scale', 'FACTOR': 0.1, 'MILESTONE': [100, 200]}, 'TRAIN': {'TRAINER': 'BiModalTrainer', 'START_EPOCH': 0, 'MAX_EPOCH': 5000, 'PRE_TRAINED_MODEL': None, 'RESUME': '', 'LOG_INTERVAL': 10, 'VALID_INTERVAL': 1, 'OUTPUT_DIR': 'results/demo/unified_frame_images/03_bimodal_resnet'}, 'TEST': {'TEST_ONLY': True, 'FULL_TEST': True, 'WEIGHT': '', 'COMPUTE_PCC': False, 'COMPUTE_CCC': False, 'SAVE_DATASET_OUTPUT': ''}}
[2024-11-11 06:24:32,042: WARNING/MainProcess] {
    "DATA":{
        "ROOT":"datasets",
        "SESSION":"talk",
        "TEST_AUD_DATA":"ChaLearn/voice_data/voice_librosa/test_data",
        "TEST_IMG_DATA":"ChaLearn/test_data",
        "TEST_IMG_FACE_DATA":"image_data/test_data_face",
        "TEST_LABEL_DATA":"ChaLearn/annotation/annotation_test.pkl",
        "TRAIN_AUD_DATA":"ChaLearn/voice_data/voice_librosa/train_data",
        "TRAIN_IMG_DATA":"ChaLearn/train_data",
        "TRAIN_IMG_FACE_DATA":"image_data/train_data_face",
        "TRAIN_LABEL_DATA":"ChaLearn/annotation/annotation_training.pkl",
        "TYPE":"frame",
        "VALID_AUD_DATA":"ChaLearn/voice_data/voice_librosa/valid_data",
        "VALID_IMG_DATA":"ChaLearn/valid_data",
        "VALID_IMG_FACE_DATA":"image_data/valid_data_face",
        "VALID_LABEL_DATA":"ChaLearn/annotation/annotation_validation.pkl",
        "VA_DATA":"va_data/cropped_aligned",
        "VA_ROOT":"datasets",
        "VA_TRAIN_LABEL":"va_data/va_label/VA_Set/Train_Set",
        "VA_VALID_LABEL":"va_data/va_label/VA_Set/Validation_Set"
    },
    "DATA_LOADER":{
        "DATASET":"",
        "DROP_LAST":true,
        "NAME":"bimodal_resnet_data_loader",
        "NUM_WORKERS":0,
        "SECOND_STAGE":{
            "METHOD":"",
            "TYPE":""
        },
        "SHUFFLE":true,
        "TRAIN_BATCH_SIZE":8,
        "TRANSFORM":"standard_frame_transform",
        "VALID_BATCH_SIZE":4
    },
    "LOSS":{
        "NAME":"mean_square_error"
    },
    "MODEL":{
        "NAME":"audiovisual_resnet",
        "NUM_CLASS":5,
        "PRETRAIN":false,
        "RETURN_FEATURE":false,
        "SPECTRUM_CHANNEL":50
    },
    "SOLVER":{
        "BETA_1":0.5,
        "BETA_2":0.999,
        "FACTOR":0.1,
        "LR_INIT":0.001,
        "MILESTONE":[
            100,
            200
        ],
        "MOMENTUM":0.9,
        "NAME":"sgd",
        "RESET_LR":false,
        "SCHEDULER":"multi_step_scale",
        "WEIGHT_DECAY":0.0005
    },
    "TEST":{
        "COMPUTE_CCC":false,
        "COMPUTE_PCC":false,
        "FULL_TEST":true,
        "SAVE_DATASET_OUTPUT":"",
        "TEST_ONLY":true,
        "WEIGHT":""
    },
    "TRAIN":{
        "LOG_INTERVAL":10,
        "MAX_EPOCH":5000,
        "OUTPUT_DIR":"results/demo/unified_frame_images/03_bimodal_resnet",
        "PRE_TRAINED_MODEL":null,
        "RESUME":"",
        "START_EPOCH":0,
        "TRAINER":"BiModalTrainer",
        "VALID_INTERVAL":1
    }
}
[2024-11-11 06:24:32,043: WARNING/MainProcess] {
    "DATA":{
        "ROOT":"datasets",
        "SESSION":"talk",
        "TEST_AUD_DATA":"ChaLearn/voice_data/voice_librosa/test_data",
        "TEST_IMG_DATA":"ChaLearn/test_data",
        "TEST_IMG_FACE_DATA":"image_data/test_data_face",
        "TEST_LABEL_DATA":"ChaLearn/annotation/annotation_test.pkl",
        "TRAIN_AUD_DATA":"ChaLearn/voice_data/voice_librosa/train_data",
        "TRAIN_IMG_DATA":"ChaLearn/train_data",
        "TRAIN_IMG_FACE_DATA":"image_data/train_data_face",
        "TRAIN_LABEL_DATA":"ChaLearn/annotation/annotation_training.pkl",
        "TYPE":"frame",
        "VALID_AUD_DATA":"ChaLearn/voice_data/voice_librosa/valid_data",
        "VALID_IMG_DATA":"ChaLearn/valid_data",
        "VALID_IMG_FACE_DATA":"image_data/valid_data_face",
        "VALID_LABEL_DATA":"ChaLearn/annotation/annotation_validation.pkl",
        "VA_DATA":"va_data/cropped_aligned",
        "VA_ROOT":"datasets",
        "VA_TRAIN_LABEL":"va_data/va_label/VA_Set/Train_Set",
        "VA_VALID_LABEL":"va_data/va_label/VA_Set/Validation_Set"
    },
    "DATA_LOADER":{
        "DATASET":"",
        "DROP_LAST":true,
        "NAME":"bimodal_resnet_data_loader",
        "NUM_WORKERS":0,
        "SECOND_STAGE":{
            "METHOD":"",
            "TYPE":""
        },
        "SHUFFLE":true,
        "TRAIN_BATCH_SIZE":8,
        "TRANSFORM":"standard_frame_transform",
        "VALID_BATCH_SIZE":4
    },
    "LOSS":{
        "NAME":"mean_square_error"
    },
    "MODEL":{
        "NAME":"audiovisual_resnet",
        "NUM_CLASS":5,
        "PRETRAIN":false,
        "RETURN_FEATURE":false,
        "SPECTRUM_CHANNEL":50
    },
    "SOLVER":{
        "BETA_1":0.5,
        "BETA_2":0.999,
        "FACTOR":0.1,
        "LR_INIT":0.001,
        "MILESTONE":[
            100,
            200
        ],
        "MOMENTUM":0.9,
        "NAME":"sgd",
        "RESET_LR":false,
        "SCHEDULER":"multi_step_scale",
        "WEIGHT_DECAY":0.0005
    },
    "TEST":{
        "COMPUTE_CCC":false,
        "COMPUTE_PCC":false,
        "FULL_TEST":true,
        "SAVE_DATASET_OUTPUT":"",
        "TEST_ONLY":true,
        "WEIGHT":""
    },
    "TRAIN":{
        "LOG_INTERVAL":10,
        "MAX_EPOCH":5000,
        "OUTPUT_DIR":"results/demo/unified_frame_images/03_bimodal_resnet",
        "PRE_TRAINED_MODEL":null,
        "RESUME":"",
        "START_EPOCH":0,
        "TRAINER":"BiModalTrainer",
        "VALID_INTERVAL":1
    }
}
[2024-11-11 06:24:32,042: INFO/MainProcess] {
    "DATA":{
        "ROOT":"datasets",
        "SESSION":"talk",
        "TEST_AUD_DATA":"ChaLearn/voice_data/voice_librosa/test_data",
        "TEST_IMG_DATA":"ChaLearn/test_data",
        "TEST_IMG_FACE_DATA":"image_data/test_data_face",
        "TEST_LABEL_DATA":"ChaLearn/annotation/annotation_test.pkl",
        "TRAIN_AUD_DATA":"ChaLearn/voice_data/voice_librosa/train_data",
        "TRAIN_IMG_DATA":"ChaLearn/train_data",
        "TRAIN_IMG_FACE_DATA":"image_data/train_data_face",
        "TRAIN_LABEL_DATA":"ChaLearn/annotation/annotation_training.pkl",
        "TYPE":"frame",
        "VALID_AUD_DATA":"ChaLearn/voice_data/voice_librosa/valid_data",
        "VALID_IMG_DATA":"ChaLearn/valid_data",
        "VALID_IMG_FACE_DATA":"image_data/valid_data_face",
        "VALID_LABEL_DATA":"ChaLearn/annotation/annotation_validation.pkl",
        "VA_DATA":"va_data/cropped_aligned",
        "VA_ROOT":"datasets",
        "VA_TRAIN_LABEL":"va_data/va_label/VA_Set/Train_Set",
        "VA_VALID_LABEL":"va_data/va_label/VA_Set/Validation_Set"
    },
    "DATA_LOADER":{
        "DATASET":"",
        "DROP_LAST":true,
        "NAME":"bimodal_resnet_data_loader",
        "NUM_WORKERS":0,
        "SECOND_STAGE":{
            "METHOD":"",
            "TYPE":""
        },
        "SHUFFLE":true,
        "TRAIN_BATCH_SIZE":8,
        "TRANSFORM":"standard_frame_transform",
        "VALID_BATCH_SIZE":4
    },
    "LOSS":{
        "NAME":"mean_square_error"
    },
    "MODEL":{
        "NAME":"audiovisual_resnet",
        "NUM_CLASS":5,
        "PRETRAIN":false,
        "RETURN_FEATURE":false,
        "SPECTRUM_CHANNEL":50
    },
    "SOLVER":{
        "BETA_1":0.5,
        "BETA_2":0.999,
        "FACTOR":0.1,
        "LR_INIT":0.001,
        "MILESTONE":[
            100,
            200
        ],
        "MOMENTUM":0.9,
        "NAME":"sgd",
        "RESET_LR":false,
        "SCHEDULER":"multi_step_scale",
        "WEIGHT_DECAY":0.0005
    },
    "TEST":{
        "COMPUTE_CCC":false,
        "COMPUTE_PCC":false,
        "FULL_TEST":true,
        "SAVE_DATASET_OUTPUT":"",
        "TEST_ONLY":true,
        "WEIGHT":""
    },
    "TRAIN":{
        "LOG_INTERVAL":10,
        "MAX_EPOCH":5000,
        "OUTPUT_DIR":"results/demo/unified_frame_images/03_bimodal_resnet",
        "PRE_TRAINED_MODEL":null,
        "RESUME":"",
        "START_EPOCH":0,
        "TRAINER":"BiModalTrainer",
        "VALID_INTERVAL":1
    }
}
[2024-11-11 06:24:32,213: WARNING/MainProcess] Test only mode
[2024-11-11 06:24:32,213: WARNING/MainProcess] Test only mode
[2024-11-11 06:24:32,213: INFO/MainProcess] Test only mode
[2024-11-11 06:24:32,213: WARNING/MainProcess] Test with model ./checkpoint.pkl
[2024-11-11 06:24:32,213: WARNING/MainProcess] Test with model ./checkpoint.pkl
[2024-11-11 06:24:32,213: INFO/MainProcess] Test with model ./checkpoint.pkl
[2024-11-11 06:24:32,460: WARNING/MainProcess] The cuda is here 
[2024-11-11 06:24:32,460: WARNING/MainProcess]  
[2024-11-11 06:24:32,460: WARNING/MainProcess] False
[2024-11-11 06:24:32,460: WARNING/MainProcess] print the model is done loading here 
[2024-11-11 06:24:32,476: WARNING/MainProcess]   0%|          | 0/1 [00:00<?, ?it/s]
[2024-11-11 06:24:32,510: WARNING/MainProcess] changes have to be done 
[2024-11-11 06:24:32,510: WARNING/MainProcess] The self.test only is given as
[2024-11-11 06:24:32,510: WARNING/MainProcess]  
[2024-11-11 06:24:32,510: WARNING/MainProcess] True
[2024-11-11 06:24:32,510: WARNING/MainProcess] ./datasets/ChaLearn/voice_data/voice_librosa/test_data/47fa15a6-4826-46d9-82c7-1e3edf4cb012.wav.npy
[2024-11-11 06:24:32,511: WARNING/MainProcess] [Errno 2] No such file or directory: './datasets/ChaLearn/voice_data/voice_librosa/test_data/47fa15a6-4826-46d9-82c7-1e3edf4cb012.wav.npy'
[2024-11-11 06:24:32,511: WARNING/MainProcess]   0%|          | 0/1 [00:00<?, ?it/s]
[2024-11-11 06:24:32,511: WARNING/MainProcess] local variable 'wav_ft' referenced before assignment
[2024-11-11 06:24:32,528: WARNING/MainProcess] Deleted file: 47fa15a6-4826-46d9-82c7-1e3edf4cb012.mp4
[2024-11-11 06:24:32,528: WARNING/MainProcess] All files deleted successfully from folder: datasets/ChaLearn/test
[2024-11-11 06:24:32,538: WARNING/MainProcess] Deleted folder: /home/ubuntu/AI_G/videos/4df48c55-88d7-44bd-a5a7-9b2f6882f65f
[2024-11-11 06:24:32,538: WARNING/MainProcess] Files are deleted in Test Folder.
[2024-11-11 06:24:32,540: WARNING/MainProcess] print result 1 is here:
[2024-11-11 06:24:32,540: WARNING/MainProcess]  
[2024-11-11 06:24:32,540: WARNING/MainProcess] None
[2024-11-11 06:24:32,557: ERROR/MainProcess] Task resultsUI.task.process_batch[6eb48b04-7adf-460c-a005-37cd1c597023] raised unexpected: DataError(1406, "Data too long for column 'status' at row 1")
Traceback (most recent call last):
  File "/home/ubuntu/AI_G/resultsUI/task.py", line 263, in process_batch
    result = json.loads(result1)  # Used to Set the Json Response
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/mysql/base.py", line 76, in execute
    return self.cursor.execute(query, args)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
MySQLdb.DataError: (1406, "Data too long for column 'status' at row 1")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/home/ubuntu/AI_G/resultsUI/task.py", line 288, in process_batch
    link_entry.save()
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/base.py", line 891, in save
    self.save_base(
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/base.py", line 997, in save_base
    updated = self._save_table(
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/base.py", line 1129, in _save_table
    updated = self._do_update(
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/base.py", line 1194, in _do_update
    return filtered._update(values) > 0
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/query.py", line 1278, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 2003, in execute_sql
    cursor = super().execute_sql(result_type)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/django/db/backends/mysql/base.py", line 76, in execute
    return self.cursor.execute(query, args)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/home/ubuntu/anaconda3/envs/s2/lib/python3.10/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
django.db.utils.DataError: (1406, "Data too long for column 'status' at row 1")
